static/docs/commands-reference/import.md

```diff
@@ -0,0 +1,62 @@
+# import
+
+Download or copy file or directory from another DVC repository into the
+<abbr>workspace</abbr>, and track changes in the remote source with DVC. Creates
+a DVC-file.
+
+> See also `dvc get` which corresponds to the first step this command performs
+> (just download the data).
+
+## Synopsis
+
+```usage
+usage: dvc import [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
+
+positional arguments:
+  url                   DVC repository URL.
+  path                  Path to data within DVC repository.
+```
+
+## Description
+
+In some cases it's convenient to add a <abbr>data artifact</abbr> from another
```

Comment:
need to come with a better articulation here. For import and get. The biggest use case is that it's easier to re-use datasets/models created in one repo within another repo. For example, one of the actual cases - datasets registry - a github repo ppl use to store and update datasets. This command provides a way to "use" this dataset from that Gtihub repo.  Will think about better message here.

Reply:
Good point. OK, I did a considerable refactoring of this in 91b4e10, what do you think?